packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -350,115 +358,59 @@ export class GarbageCollector implements IGarbageCollector {
 			this.tombstones = Array.from(baseSnapshotData.tombstones);
```

Comment:
Nit: The function comment seems out of date, at first it sounds like it's only about Tombstones.

Reply:
More importantly it should be updated to explain that it's doing full init including the core GC Data, not just Tombstoned/Deleted nodes.